apps/ledger-live-mobile/src/screens/SyncOnboarding/companionSteps/BackupStep.tsx

```diff
@@ -0,0 +1,232 @@
+          state: {
+            routes: [
+              {
+                name: NavigatorName.Main,
+              },
+            ],
+          },
+        },
+      ],
+    });
+
+    //Open Live App Recover
+    navigation.navigate(NavigatorName.Base, {
+      screen: ScreenName.Recover,
+      params: {
+        device,
+        redirectTo: "activate",
+        // platform: "protect-staging", // TODO: remove this, only for testing in debug
+        platform: servicesConfig?.params?.protectId, // TODO: reenable this
+        date: new Date().toISOString(), // adding a date to reload the page in case of same device restored again
+      },
+    });
+  }, [device, dispatchRedux, navigation, servicesConfig?.params?.protectId]);
+
+  return isOpened ? (
+    <Flex mt={3} rowGap={space[3]}>
+      <ChoiceText mb={3}>{t("syncOnboarding.backup.backupChoice.description")}</ChoiceText>
+      <Button
+        size="small"
+        type="main"
+        title={t("syncOnboarding.backup.backupChoice.tryCta")}
+        onPress={navigateToRecover}
+        event="button_clicked"
+        eventProperties={{ button: "Try for free", flow: "Device onboarding" }}
+      />
+      <Button
+        size="small"
+        type="main"
+        outline
+        title={t("syncOnboarding.backup.backupChoice.redeemCodeCta")}
+        onPress={navigateToRecover}
+        event="button_clicked"
+        eventProperties={{ button: "Apply your redeem code", flow: "Device onboarding" }}
+      />
+      <Stories
+        instanceID={StorylyInstanceID.backupRecoverySeed}
+        StoryGroupItemComponent={VideoLink}
+        storyItemTextColor="#00000000"
+        storyHeaderIconIsVisible={false}
+        noLoadingPlaceholder
+      />
+    </Flex>
+  ) : null;
+};
+
+const KeepOnPaperBody: React.FC<ChoiceBodyProps> = ({ isOpened, onPressKeepManualBackup }) => {
+  const { t } = useTranslation();
+  return isOpened ? (
+    <Flex mt={3}>
+      <ChoiceText mb={3}>{t("syncOnboarding.backup.manualBackup.description")}</ChoiceText>
+      <Button
+        type="main"
+        size="small"
+        onPress={onPressKeepManualBackup}
+        event="button_clicked"
+        eventProperties={{ button: "Keep it on Paper", flow: "Device onboarding" }}
+      >
+        {t("syncOnboarding.backup.manualBackup.cta")}
+      </Button>
+    </Flex>
+  ) : null;
+};
+
+const choices: Choice[] = [
+  {
+    id: "backup",
+    getTitle: selected =>
+      selected
+        ? "syncOnboarding.backup.backupChoice.titleSelected"
+        : "syncOnboarding.backup.backupChoice.title",
+    icon: <Icons.ShieldCheck size={"S"} color="primary.c80" />,
+    tag: "syncOnboarding.backup.backupChoice.tag",
+    tagType: "color",
+    body: BackupBody,
+  },
+  {
+    id: "keep_on_paper",
+    getTitle: () => "syncOnboarding.backup.manualBackup.title",
+    icon: <Icons.Note size={"S"} color="neutral.c80" />,
+    body: KeepOnPaperBody,
+  },
+];
+
+const BackupStep: React.FC<Props> = props => {
```

Comment:
Same with [this comment](https://github.com/LedgerHQ/ledger-live/pull/6645/files#r1574936953).